Resources/Prototypes/_Funkystation/Reagents/medicine.yml

```
@@ -0,0 +1,27 @@
+- type: reagent
+  id: Deunomia
+  name: reagent-name-deunomia
+  group: Medicine
+  desc:  reagent-desc-deunomia
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)